package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "jsgantt-improved",
+  "version": "2.0.2",
+  "description": "jsgantt-improved",
+  "main": "dist/index.js",
+  "scripts": {
+    "start": "node server.js",
+    "dev": "nodemon server.js",
+    "build": "tsc",
+    "watch": "onchange 'src/**' '*.*' -- npm run build",
+    "watch:dist": "onchange 'src/**' '*.ts' -- npm run dist",
+    "watch:test": "onchange 'src/**/*.ts' '*.ts' -- npm run test",
+    "test": "node node_modules/.bin/protractor protractor.conf.js",
+    "test-unit": "mocha -r ts-node/register test/**.ts",
+    "browserify": "browserify dist/index.js --standalone JSGantt > jsgantt.js",
+    "dist": "npm run build && npm run browserify && cp src/jsgantt.css dist/ && cp jsgantt.js dist/ && cp jsgantt.js docs/ && cp src/jsgantt.css demo/ && echo 'DIST finished'",
+    "publish-npm": "npm run dist && npm publish",
+    "demo": "nodemon server.js",
+    "demo-full": "npm run dist && npm run demo",
+    "e2e-prepare": "npm i -g webdriver-manager && webdriver-manager update && ./node_modules/protractor/node_modules/webdriver-manager/bin/webdriver-manager update"
```

Comment:
based on this issue [here](https://github.com/angular/webdriver-manager/issues/216#issuecomment-293650564), we should just do `webdriver-manager update --gecko false` to see if improves build

Reply:
what u mean about improve?

Reply:
because of this error during build: https://travis-ci.com/jsGanttImproved/jsgantt-improved/builds/92611601

Reply:
it because we changed to docs...